Chummer/Classes/clsOptions.cs

```diff
@@ -622,16 +633,16 @@ public static bool UseLogging
         /// <summary>
         /// Whether or not the app should use logging.
         /// </summary>
-        public static bool UseLoggingApplicationInsights
+        public static EnumUseLoggingApplicationInsights UseLoggingApplicationInsights
         {
-            get => _blnUseLoggingApplicationInsights;
+            get => (EnumUseLoggingApplicationInsights)Enum.Parse(typeof(EnumUseLoggingApplicationInsights), _enumUseLoggingApplicationInsights.ToString());
```

Comment:
Are you converting the enum to a string, parsing it again, then casting it to its original type?
Is there a reason `get => _enumUseLoggingApplicationInsights;` does not work?